workbench/src/renderer/static/about.js

```diff
@@ -0,0 +1,19 @@
+import { ipcMainChannels } from '../../main/ipcMainChannels';
+import { handleClick } from './handlers';
+
+const { ipcRenderer } = window.Workbench.electron;
```

Comment:
Not sure if this is really a rule, but I associate `static/` with containing things that aren't code.

Reply:
You're probably right. This was probably leftover from when this folder just held html documents. I truly don't know what the right naming conventions are though. Maybe this folder should be `src/menubar`? or `src/renderer/menubar`

Reply:
I'm putting these .js files in `src/renderer/menubar/` and leaving the one .png here in `static`